Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -468,7 +474,16 @@ public class EOSLobbyManager : IEOSSubManager
         public delegate void OnMemberUpdateCallback(string LobbyId, ProductUserId MemberId);
 
         private List<OnMemberUpdateCallback> MemberUpdateCallbacks;
-        private List<Action<LobbyChangedEvent>> LobbyChangeCallbacks;
+
+
+        public delegate void LobbyChangedEventHandler(string lobbyId, LobbyChangeType typeOfChange);
+
+        /// <summary>
+        /// Event that is run whenever the local user's relationship to a Lobby has been changed.
+        /// Indicates the Lobby that the change relates to.
+        /// </summary>
+        public event LobbyChangedEventHandler OnLobbyChanged;
```

Comment:
This should be named, `LobbyChanged`. The fact that it is an event is the indicator that it is an action / event / verb. This allows the functions that _subscribe_ to it (ala the `+=` operator) to be named `OnLobbyChange` without confusion.

Reply:
I should not that we are not doing this consistently throughout the project, but that this is the Correct™ way to do it from my understanding of industry conventions.

Reply:
I did just find this: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-events
There are other aspects of this we do not follow (and probably should), but the one regarding what to name the event is consistent with my recommendation I believe.

Reply:
Here are three examples within our code base of us using different naming conventions:

https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/7806e659f285992b38a17395a4495f56dcb8d078/Assets/Scripts/StandardSamples/Steam/SteamManager.cs#L71

https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/7806e659f285992b38a17395a4495f56dcb8d078/Assets/Scripts/StandardSamples/Services/EOSService.cs#L47

https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/7806e659f285992b38a17395a4495f56dcb8d078/Assets/Scripts/StandardSamples/Discord/DiscordManager.cs#L177

Reply:
For now, dropped `EventHandler` from the name.